docs/_scripts/advisory.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Get advisory
+echo '{
+    "from": "nobody@redhat.com",
+    "issued": "2014-09-24 00:00:00",
```

Comment:
updated date is one of the required fields, I'd remove `from` and set the updated field instead